tools/c7n_azure/c7n_azure/output.py

```diff
@@ -43,16 +43,17 @@ class AzureStorageOutput(DirectoryOutput):
     def __init__(self, ctx, config=None):
         super(AzureStorageOutput, self).__init__(ctx, config)
         self.log = logging.getLogger('custodian.output')
-        self.date_path = datetime.datetime.now().strftime('%Y/%m/%d/%H')
+
+        # folder structure Year/Month/Day/Hour/Second
+        self.date_path = datetime.datetime.now().strftime('%Y/%m/%d/%H/%M/%S')
```

Comment:
fwiw, `-s` on the cli now supports arbitrary date formatting via pyformat (`now` is the datetime variable for interpolation).

Reply:
Is this documented? I couldn't find an example

Reply:
not really.. its fairly recent as part of the output refactor

Reply:
I added an example to our docs